site/src/pages/release-notes/1.26.0.mdx

```diff
@@ -0,0 +1,216 @@
+---
+date: 2023-10-23
+---
+
+## 🌟 New features
+
+- **Context Path Support**: You can now set services/decorators under a context path fluently. #3591 #4802 #5037 #5157
+  ```java
+  Server.builder()
+        .contextPath("/v1", "/v2")
+        .service(myService) // served under "/v1" and "/v2"
+        .and()
+        .virtualHost("foo.com")
+        .contextPath("/v3")
+        .service(myService) // served by virtual host "foo.com" under "/v3"
+        .build();
+  ```
+- **Easier Request Scoping for Kotlin**: You can now easily propagate an Armeria <type://RequestContext> in Kotlin using
+  <type://ContextAwareExecutor#asCoroutineDispatcher()>. #5171
+  ```kotlin
+  withContext(ctx.eventLoop().asCoroutineDispatcher()) {
+    assert RequestContext.current() == ctx
+    ...
+  }
+  ```
+- **gRPC Exception Handler Annotation**: You can now easily handle an exception that is raised from a gRPC service using
+  <type://@GrpcExceptionHandler> annotation. #5046
+  ```java
+  @GrpcExceptionHandler(MyGrpcExceptionHandlerFunction.class)
+  @Override
+  public void unaryCall(
+    SimpleRequest request,
+    StreamObserver<SimpleResponse> responseObserver) {
+      ...
+  }
+
+  class MyGrpcExceptionHandlerFunction implements GrpcExceptionHandlerFunction {
+    @Override
+    public Status apply(RequestContext ctx, Throwable cause, Metadata metadata) {
+      if (unauthenticatedExcpetion(cause)) {
+        return Status.UNAUTHENTICATED;
+      }
+      ...
+    }
+  }
+  ```
+- **StreamMessage finalizer**: You can now produce the last value for a <type://StreamMessage> when it completes exceptionally using
```

Comment:
The finalizer will be invoked when `StreamMessage` completes exceptionally or successfully.